### PR TITLE
Improve demo

### PR DESCRIPTION
**Pull request by blasten**
screen shot 2015-06-04 at 12 36 38 pm

### CODE REVIEW

**Comment by notwaldorf** [comment]
I thiiiiiiink this demo would actually work best as a 3 column demo page (see the `paper-checkbox` demo). Sorry to be so picky! 😳

**Comment by blasten** [comment]
@notwaldorf what about now?  I found an accessibility issue in `Polymer.IronMenubarBehavior`

File: demo/index.html [hunk 1]
```diff
@@ -19,82 +19,94 @@
 
   <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
 
+  <link rel="import" href="../../paper-styles/paper-styles.html">
+  <link rel="import" href="../../paper-styles/demo-pages.html">
```
Comment (notwaldorf) [inline demo/index.html:23]: YOu don't need `paper-styles` anymore

File: demo/index.html [hunk 2]
```diff
@@ -19,82 +19,94 @@
...
-      <li>item 0</li>
-      <li>item 1</li>
-      <li>item 2</li>
-      <li>item 3</li>
-      <li>item 4</li>
-    </simple-menubar>
-  </section>
+  <div class="horizontal center-justified layout">
+    <div>
+      <h3>Simple menu</h3>
```
Comment (notwaldorf) [inline demo/index.html:62]: These are `h4`'s in the other demos

**Comment by notwaldorf** [comment]
I hadn't really got a chance to review this, but LGTM % nits.

**Pull request merged**